docs/tutorials/creating-a-runtime-module.md

```diff
@@ -51,184 +73,286 @@ You should be able to successfully compile this template module with:
+need to tell the module to only build its `std` feature when the runtime itself
+does, as followed:
 
-First we need to add our newly-created crate as a dependency in the node runtime's `Cargo.toml`.
+**`my-node/runtime/Cargo.toml`**
+
+```TOML
+#--snip--
 
-```toml
 [dependencies.test_module]
 default_features = false
-path = "../../test_module"
-```
+path = "../../my-module"
 
-> You **must** set `default_features = false` so that your runtime will successfully compile to wasm.
-
-And just as before, we need to tell the module to only build its `std` feature when the runtime itself does.
-```toml
+# toward the bottom
+[features]
+default = ['std']
 std = [
-  ...
-  'test_module/std',
+    'test_module/std',
+    #--snip--
 ]
 ```
 
-Next we'll update `runtime/lib.rs` in `modular-chain` to actually use our new runtime module. The lines that bring in the template module that is coded directly in the node runtime are still there, so we just need to update them in three places.
+> You **must** set `default_features = false` so that your runtime will
+successfully compile to wasm.
+
+Next we'll update `runtime/src/lib.rs` to actually use our new runtime module.
+We will update the trait implementation with our `test_module` and add it in
+our runtime construction macro.
+
+**`my-node/runtime/src/lib.rs`**
 
-1. Remove `mod template` as we've already brought that dependency in via `Cargo.toml`
-2. Update the trait implementation
 ```rust
-/// Used for the module test_module
+// already existed
+impl template::Trait for Runtime {
+  type Event = Event;
+}
+
+// add the following code block
 impl test_module::Trait for Runtime {
 	type Event = Event;
 }
-```
 
-3. Update `construct_runtime`
-```rust
-TestModule: test_module::{Module, Call, Storage, Event<T>},
+//...
+construct_runtime!(
+  pub enum Runtime where
+    Block = Block,
+    NodeBlock = opaque::Block,
+    UncheckedExtrinsic = UncheckedExtrinsic
+  {
+    //...
+    // add the following line
+    TestModule: test_module::{Module, Call, Storage, Event<T>},
+  }
+);
 ```
 
-## Appreciate your Work
-At this point you have the template module packaged up as it's own crate, and included in your `modular-chain` runtime.
+### Appreciate your Work 😊
+
+At this point you have the module packaged up as it's own crate, and included in
+your node runtime. Compile and run your node with:
 
-Build the `modular-chain` node with
 ```bash
-./scripts/build.sh
+cd my-playground-folder/my-node
+# compile your node
 cargo build --release
+# purge any existing chain
+cargo run --release -- purge-chain --dev
+# start the node
+cargo run --release -- --dev
 ```
 
-And run your node with
-```bash
-# Clear the data directory in case you've done this before
-./target/release/modular-chain purge-chain --dev
+Finally, start the [Polkadot apps UI](https://polkadot.js.org/apps/#/explorer)
+to confirm that the module is working as expected. In the apps UI, you will need
```

Comment:
You could change the url to `https://polkadot.js.org/apps/#/explorer?rpc=ws://127.0.0.1:9944`

If you want to avoid telling these instructions.